icu4c/source/i18n/unicode/unumberformatter.h

```diff
@@ -147,6 +147,28 @@ typedef enum UNumberUnitWidth {
      */
             UNUM_UNIT_WIDTH_ISO_CODE,
 
+    /**
```

Comment:
In public header files, you need to wrap `@draft` APIs with `U_HIDE_DRAFT_API`.  Search for examples.